### ISSUE TITLE
Keyboard Focus Issue

### ISSUE THREAD

**Issue by Brawlboxgaming** [Epik95mkw/kmp-editor#8]
- After typing a value in the selection section, the keyboard focus needs to change before doing anything on the main view i.e. Alt+Click+Dragging a point does not work until the keyboard focus has been changed.
